Lib/defconQt/featureTextEditor.py

```diff
@@ -17,11 +17,11 @@ def __init__(self, font=None, parent=None):
         self.editor = FeatureTextEditor(self.font.features.text, self)
         self.resize(600, 500)
 
-        fileMenu = QMenu("&File", self)
-        fileMenu.addAction("&Save…", self.save, QKeySequence.Save)
+        fileMenu = QMenu(self.tr("&File"), self)
+        fileMenu.addAction(self.tr("&Save") + "…", self.save, QKeySequence.Save)
         fileMenu.addSeparator()
-        fileMenu.addAction("&Reload From Disk", self.reload)
-        fileMenu.addAction("E&xit", self.close, QKeySequence.Quit)
+        fileMenu.addAction(self.tr("&Reload From Disk"), self.reload)
+        fileMenu.addAction(self.tr("&Close"), self.close, QKeySequence.Quit)
```

Comment:
The naming is a bit unfortunate. "File" -> "Exit" suggests that it exits the program, or at least the file. "File" -> "Close" is still unfortunate, but a bit better imo.